packages/flutter_version_manager/lib/src/ui/widgets/mandatory_update_dialog.dart

```diff
@@ -0,0 +1,97 @@
+// ignore_for_file: avoid_positional_boolean_parameters
```

Comment:
do not ignore positional boolean parameters, use enums or other state indicators for more descriptive parameters.